skt/executable.py

```diff
@@ -460,6 +461,50 @@ def cmd_report(cfg):
     reporter.report()
 
 
+def cmd_console_check(cfg):
+    """
+    Check the console logs for any traces.
+
+    Args:
+        cfg: A dictionary of skt configuration.
+    """
+    remove_oldresult(cfg.get('output_dir'), 'console_check.')
+    console_result_path = join_with_slash(cfg.get('output_dir'),
+                                          'console_check.result')
+    console_report_path = join_with_slash(cfg.get('output_dir'),
+                                          'console_check.report')
+
+    if not cfg.get('krelease') or not cfg.get('console'):
+        raise Exception('<krelease> or <console-url> parameter missing!')
+
+    console_logs = []
+    trace_list = []
+
+    for console_path_or_url in cfg.get('console'):
+        console_log = skt.console.ConsoleLog(cfg.get('krelease'),
+                                             console_path_or_url)
+
+        console_logs.append(console_log)
```

Comment:
Why do we keep those around?

Reply:
I wanted to be able to attach the parsed out log in this stage if someone was to run the stage separately, outside of skt. Since we don't have the input/output directory schema fully working yet, that part is not done. If you think that's a bad idea, I can remove this preparation.

Reply:
I think it is better not to add unused code, in general. Every time somebody reads this they would wonder what's going on, until the code using this is added. It's not a big deal here, so leave it if you feel you'll add something to use it soon, but in general I think it's best to avoid it.